src/main/resources/blogs/bloggers.json

```diff
@@ -834,10 +834,10 @@
       "twitter": "@DevolutionTech"
     },
     {
-      "bookmarkableId": "szymon-bilinski",
-      "name": "Szymon Biliński",
+      "bookmarkableId": "szymon-jednac",
```

Comment:
I don't really think that bookmarkableId should be changed. It is persisted in db with your previous blog entries, so changing it would make it inaccessible.

Reply:
It might mess only one or two links in all newsletter issues so I think we can live with that :)

Reply:
I was wondering about this myself. Anyway, thank you for including the change!